fastmri_recon/models/utils/fourier.py

```diff
@@ -17,6 +17,10 @@ def tf_unmasked_adj_op(x, idx=0):
     scaling_norm = tf.dtypes.cast(tf.math.sqrt(tf.dtypes.cast(tf.math.reduce_prod(tf.shape(x)[1:3]), 'float32')), x.dtype)
     return scaling_norm * tf.expand_dims(fftshift(ifft2d(ifftshift(x[..., idx], axes=[1, 2])), axes=[1, 2]), axis=-1)
 
+def tf_unmasked_adj_op_non_cartesian(x, idx=0):
```

Comment:
this is not what it says. It doesn't use non cartesian, probably a legacy

Reply:
Actually I had a question here. I think your code on top isnt right , and I could not isolate how [1, 2] axes could work.
Generally the kspace that we get from data set is as:
[:, Nz, Nx, Ny, :]

I forgot to mention this to you. I didnt have enough time to debug into shapes for cartesian model

Reply:
Yes the ops above work well only for images that are `batch x height x width x channels`, so not very general.

However it works for the use cases it serves, so I think it's fine for now. This new introduced function is not non-cartesian though, so I think it should be removed.